Update docs/guides to reference MFA (#24689)
fleetdm/fleet: articles/fleetctl.md

Once your local context is configured, you can use fleetctl normally.

> ### Log in with SAML (SSO) authentication or MFA

Author: Seems reasonable, and I just confirmed that we don't have any link anchors pointing at this so we aren't breaking anything by swapping this header (should've checked when making the initial change).

> #### Users with sign sign-on (SSO) or email two-factor authentication (2FA)

Reviewer: ```suggestion
#### Log in with single sign-on (SSO) or email two-factor authentication (2FA)
```

Author: Do we want "Users" or "Log in" here? The change to "Users" was from @noahtalerman

Reviewer: oh never mind, I see this is a subheading

> Users that authenticate to Fleet via SSO, or who have email MFA set up, should retrieve their API token from the UI and manually set it in their fleetctl configuration (instead of logging in via `fleetctl login`).

Reviewer 2: Updating language for clarity. Let me know what you think.

> Users that authenticate to Fleet via SSO, or who have email MFA set up, should retrieve their API token from the UI and manually set it in their fleetctl configuration (instead of logging in via `fleetctl login`).

Author: That works.

> Users that login to Fleet with SSO or email 2FA can't login with `fleetctl login`. Instead they can retrieve their API token from the UI and manually set it in their fleetctl configuration (instead of logging in via `fleetctl login`).

Reviewer: "login" - noun, "log in" - verb
```suggestion
Users that log into Fleet with SSO or email 2FA can't log in with `fleetctl login`. Instead they can retrieve their API token from the UI and manually set it in their fleetctl configuration (instead of logging in via `fleetctl login`).
```
